modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java

```diff
@@ -372,6 +374,15 @@ public Integer getStatus() {
 		return _status;
 	}
 
+	public ActionURL getSwapConfigurationURL() {
```

Comment:
Could we return a generic PortletURL instead?

Reply:
Of course we could, but why would we want to do so? IMHO the method is more explicit like it is, and there's at least an example here `com.liferay.segments.context.vocabulary.internal.display.context.SegmentsContextVocabularyConfigurationDisplayContext#getActionURL`. I'm totally open to change it, just want to be sure I get it correctly

Reply:
That makes sense :)